src/EditorFeatures/CSharpTest/InvertIf/InvertIfTests.cs

```diff
@@ -1291,5 +1291,174 @@ int M()
     }
 }", parseOptions: CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp9));
         }
+
+        [Fact, WorkItem(63311, "https://github.com/dotnet/roslyn/issues/63311")]
+        public async Task TestLiftedNullable_GreaterThan()
+        {
+            await TestInRegularAndScriptAsync(
+                """
+                class C
+                {
+                    void M(int? p)
+                    {
+                        [||]if (p > 10)
```

Comment:
Maybe create a `[Theory]` and use operator as a test parameter?

Reply:
I decided against this originally based on looking at nearby tests.